File: hw/elbay/u3/hw25TemplateMethod/hw25TemplateMethod/java/src/main/java/model/PastaDish.java
package model;

public class PastaDish extends MealPreparation {
    @Override
    public void cook() {
        System.out.println("Boiling pasta and simmering sauce");
    }

    @Override
    public void addGarnish() {
        System.out.println("Adding grated parmesan and basil");
    }
}

File: hw/elbay/u3/hw25TemplateMethod/hw25TemplateMethod/js/model/PastaDish.js
const MealPreparation = require("./MealPreparation");

class PastaDish extends MealPreparation {
    cook() {
        console.log("Boiling pasta and simmering sauce");
    }

    addGarnish() {
        console.log("Adding grated parmesan and basil");
    }
}

module.exports = PastaDish;
